app/helpers/mail_helper.rb

```diff
@@ -0,0 +1,47 @@
+module MailHelper
+
+  APP_GUIDANCE_PAGE = 'https://intranet.justice.gov.uk/peoplefinder'.freeze
+
+  def browser_warning
+    content_tag(:p, class: 'browser-warning') do
+      mailer_t(:browser_warning)
+    end
+  end
+
+  def easy_copy_link_to url:
```

Comment:
no problem. Although use or not of parentheses is pretty inconsistent all over the app :nauseated_face: